hack/test-cmd.sh

```diff
@@ -238,6 +238,9 @@ fi
 PATH="${KUBE_OUTPUT_HOSTBIN}":$PATH
 
 kube::log::status "Checking kubectl version"
+
+# Note that this fails on a dirty commit.  To avoid that from happening, .gitignore
```

Comment:
failure of `kubectl version` is because the generated source makes the commit hash dirty.  this was tricky to track down ! :)